ui/apps/platform/src/Containers/Risk/RiskTablePanel.tsx

```diff
@@ -85,7 +76,7 @@ function RiskTablePanel({
 
         const hasSearchFilters = Object.keys(searchFilter).length > 0;
         setIsViewFiltered(hasSearchFilters);
-    }, [searchFilter, sortOption, currentPage, shouldHideOrchestratorComponents]);
+    }, [searchFilter, sortOption, page, shouldHideOrchestratorComponents]);
```

Comment:
The absence of `perPage` made me curious to expand and see this is `useDeepCompareEffect` hook.

Because only 5 occurrences, what is the probability that a future step will replace it here in Risk?